src/input/mod.rs

```diff
@@ -2596,7 +2636,7 @@ mod tests {
 
         mods = Default::default();
         let filter = close_key_event(&mut suppressed_keys, mods, false);
-        assert!(matches!(filter, FilterResult::Intercept(None)));
+        assert!(matches!(filter, FilterResult::Forward));
```

Comment:
This must not change. This is what prevents the client from getting the release of a compositor key, for which it did not see the press.